CardCollectionView/Source/Controller/View Controller/DetailViewController.swift

```diff
@@ -21,7 +21,7 @@ class DetailViewController: UIViewController {
     
     // MARK: - Status Bar
     
-    override func preferredStatusBarStyle() -> UIStatusBarStyle {
-        return .Default
-    }
+//    override func preferredStatusBarStyle() -> UIStatusBarStyle {
```

Comment:
`preferredStatusBarStyle` is now implemented as a property in Swift 3, this should be reflected instead of commenting out the functionality e.g.: ```override var preferredStatusBarStyle: UIStatusBarStyle {
        return .default
    }```